draft-ietf-quic-transport.md

```diff
@@ -2720,17 +2720,19 @@ connection in a recoverable state, the endpoint can send a RESET_STREAM frame
 ({{frame-reset-stream}}) with an appropriate error code to terminate just the
 affected stream.
 
-RESET_STREAM MUST be instigated by the protocol using QUIC.  RESET_STREAM
-carries an application error code.  Only the application protocol is able to
+RESET_STREAM MUST be instigated by the application protocol that uses QUIC.
```

Comment:
It's more like a transport MUST NOT - surely an application protocol can get through a connection without sending a RESET.

Reply:
STOP_SENDING instigates RESET_STREAM, so this is a bit of a confusing statement.

Reply:
```suggestion
RESET_STREAM MUST only be instigated by the application protocol that uses QUIC.
```
@mikkelfj is right -- in isolation, this reads like a requirement on the application, not a restriction on everyone else.